cirq/work/zeros_sampler.py

```diff
@@ -13,27 +13,16 @@
 # limitations under the License.
 
 import abc
-from typing import (Dict, List, TYPE_CHECKING)
+from typing import Dict, List
 
 import numpy as np
 
 from cirq import work, study, protocols
 
-if TYPE_CHECKING:
-    import cirq.google
-
 
 class ZerosSampler(work.Sampler, metaclass=abc.ABCMeta):
     """A dummy sampler for testing. Immediately returns zeroes."""
 
-    def __init__(self, gate_set: 'cirq.google.SerializableGateSet' = None):
-        """
-        Args:
-            gate_set: `SerializableGateSet`. If set, sampler will validate that
-                all gates in the circuit are from the given gate set.
-        """
-        self.gate_set = gate_set
-
     def run_sweep(
```

Comment:
I am fine with getting rid of validation altogether, but one alternative I just thought of was to add an optional device parameter.  That could also validate the circuit in a more natural way.

Reply:
OK added device validation.  Could also just do this by adding it to a circuit.